src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -197,15 +198,12 @@ void WebRTCTransportProviderServer::RemoveSession(uint16_t sessionId)
     }
 }
 
-CHIP_ERROR WebRTCTransportProviderServer::ValidateExistingSession(HandlerContext & ctx, uint16_t sessionId,
-                                                                  WebRTCSessionStruct *& outSessionPtr)
+WebRTCSessionStruct * WebRTCTransportProviderServer::CheckForMatchingSession(HandlerContext & ctx, uint16_t sessionId)
 {
-    outSessionPtr = FindSession(sessionId);
+    WebRTCSessionStruct * outSessionPtr = FindSession(sessionId);
```

Comment:
```suggestion
    WebRTCSessionStruct * session = FindSession(sessionId);
```
is probably more readable.